### ISSUE TITLE
关于beta的问题

### ISSUE THREAD

**Issue by PureMmm** [gangsterless/pytorch-softmax-splatting#2]
你好，看到你代码是用的中文注释我就用中文问了。我看到你代码里面有用两个beta超参数，但是我注意到你用了unet网络学习Z，最后乘以beta，这个beta就是用的初始化一个数字而没有去学习，所以你的确是没有学习这个参数对吗？我还注意到unet网络比较大，是不是按照论文中说的其实没必要使用unet，只需要去去学习beta这个参数可以达到相同的效果。期待你的回复，非常谢谢啦！因为最近我也在做这篇论文的相关工作，我是一个小菜鸟，希望能得到你的回答。或者可以留一个联系方式或邮箱之类的吗以方便之后一起交流。再次表示我的感谢！

**Comment by gangsterless** [comment]
ok 老哥你可以加我qq
935309780
